content/3_Silver/2P.problems.json

```diff
@@ -84,10 +84,9 @@
       "source": "CF",
       "difficulty": "Easy",
       "isStarred": false,
-      "tags": ["Two Pointers"],
+      "tags": ["2P", "Sets"],
```

Comment:
Sets is not really a tag.

Reply:
If this problem was heavily dependent on sets, it would fall under "Data Structures" category imo